app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx

```diff
@@ -38,13 +38,14 @@ const QueryEditor: FC<QueryEditorProps> = ({
   const wrapperEl = useRef<HTMLUListElement>(null);
 
   const openAutocomplete = useMemo(() => {
-    return !(!autocomplete || downMetaKeys.length || query.length < 2 || !focusField);
+    const words = query.split(/_|\.|\s+|=|\{/).length;
```

Comment:
The `_` and `.` can present in metric name. See the full list of characters, which can present in metric name at  https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels . VictoriaMetrics additionally supports also `.` character, which is frequently used in [Graphite metrics](https://stackoverflow.com/questions/17503994/correct-use-of-graphite-metric-names), which are [accepted by VictoriaMetrics](https://docs.victoriametrics.com/#how-to-send-data-from-graphite-compatible-agents-such-as-statsd).

Reply:
Changed the logic